Wrapping/R/CMakeLists.txt

```diff
@@ -91,6 +91,33 @@ if(NOT SimpleITK_FORBID_DOWNLOADS)
     )
 
 endif()
+# Download tar of Rd files
+if(NOT SimpleITK_FORBID_DOWNLOADS)
+  include(sitkExternalData)
```

Comment:
yes, I meant one if statement. You can add multiple commands to the PRE_BUILD step of the target to handle the two different types.